Wikipedia/Localizations/ar.lproj/Localizable.strings

```diff
@@ -506,6 +506,11 @@
 "icon-shortcut-random-title" = "مقالة عشوائية";
 "icon-shortcut-search-title" = "البحث في ويكيبيديا";
 "image-gallery-unknown-owner" = "المؤلف غير معروف.";
+"import-shared-reading-list-survey-prompt-button-cancel" = "ليس الآن";
+"import-shared-reading-list-survey-prompt-button-take-survey" = "المشاركة في الاستطلاع";
+"import-shared-reading-list-survey-prompt-footer" = "\"اطلع على $1بيان الخصوصية$2 الخاص بنا. الاستطلاع مدعوم من قِبل جهة خارجية. اطلع على $3سياسة الخصوصية$4 الخاصة بهم.\"";
```

Comment:
Here's an example of the included in string quotation marks.

Reply:
Yeah I know what you mean - originally I stripped them out as I was copying them over, then second-guessed myself. The French and German marks made me think it was deliberate, but I'll confirm. Thanks for checking!

Reply:
Update - I confirmed with Android that we should remove these.

Reply:
👍🏽 will merge when tests pass!